modules/home-environment.nix

```diff
@@ -552,6 +552,9 @@ in
 
             cp ${activationScript} $out/activate
 
+            mkdir $out/bin
+            ln -s $out/activate $out/bin/home-manager-generation
```

Comment:
```suggestion
            ln -s $out/activate $out/bin/activate
```

To have a similar API to NixOS.

Reply:
When I tested it, nix run only tries to run `$out/bin/<package-name>` (that's why `nix run nixpkgs#neovim` doesn't work).
So if the symlink is moved to a different location you can't run it automatically anymore.

Reply:
see NixOS/nix#4498

Reply:
Right, I forgot about this peculiarity of `nix run`. It's good to know `meta.mainProgram` exists now.
I usually always do `nix shell nixpkgs#something -c some-program`